lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -18,6 +18,7 @@
 - limit-whitelist
 - max-body-size
 - owasp-modsecurity-crs
+- owasp-modsecurity-custom-rules
```

Comment:
You'll need to increment the version of the library whenever you make changes so that charmcraft can identify those

Reply:
Not quite sure, like this?

```python
# Increment this major API version when introducing breaking changes
LIBAPI = 2

# Increment this PATCH version before using `charmcraft publish-lib` or reset
# to 0 if you are raising the major API version
LIBPATCH = 0
```

Reply:
LIBPATCH increased, thanks.